webapp/src/components/boardsSwitcherDialog/boardSwitcherDialog.scss

```diff
@@ -20,4 +21,14 @@
         text-overflow: ellipsis;
         white-space: nowrap;
     }
+
+    .resultTitle {
+        max-width: 60%;
+    }
+
+    .teamTitle {
+        right: auto;
+        margin-left: auto;
+        color: rgba(var(--center-channel-color-rgb), 0.56);
```

Comment:
Did you really intend `0.56`?

Reply:
Yes, it's intentional 😛  . It's referenced from the same thing happening in mattermost webapp here - https://github.com/mattermost/mattermost-webapp/blob/master/sass/components/_suggestion-list.scss#L40